addons/l10n_ua/data/menuitem_data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="account_reports_ua_statements_menu"
```

Comment:
I'm not sure I understand why you added a menu seeing that there is nothing under it? Is there perhaps an enterprise branch with reports that needs to be reviewed along this PR?